python/sqlalchemy/demo/db/base.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+#
+#   Author  :   XueWeiHan
```

Comment:
We recommend not setting author information in code files, as all changes are already recorded by Git.